### PR TITLE
commit 1

### PR DESCRIPTION
**Pull request by Dimoshua**

### CODE REVIEW

**Comment by Dimoshua** [comment]
I am ready

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://Dimoshua.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://Dimoshua.github.io/layout_html-form/report/html_report/)
```
Comment (SanyaBratashchuk) [inline readme.md:4]: Do not forget to add this links to the PR description

File: src/index.html [hunk 2]
```diff
@@ -4,14 +4,224 @@
...
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input type="text"
+               id  ="surname"
+               name="Personal_information"
+               autocomplete="off"
+               required
+        >
```
Comment (SanyaBratashchuk) [inline src/index.html:29]:
```suggestion
        <input 
          type="text"
          id  ="surname"
          name="Personal_information"
          autocomplete="off"
          required
        >
```
Fix it everywhere

File: src/index.html [hunk 3]
```diff
@@ -4,14 +4,224 @@
...
+
+        <div><br></div>
```
Comment (SanyaBratashchuk) [inline src/index.html:31]:
```suggestion
```
Remove it everywhere you just can apply `display:block` for input's wrappers

File: src/index.html [hunk 4]
```diff
@@ -4,14 +4,224 @@
...
+               required
+        >
+        <label for="nocats">Yes</label>
+        <input type="radio"
+               id  ="nocats"
+               name="Interesting_fact"
+               value="No"
+               required
+        >
+        <label for="radio">No</label>
```
Comment (SanyaBratashchuk) [inline src/index.html:116]: It doesn't work properly

File: src/style.css [hunk 5]
```diff
@@ -1 +1,4 @@
 /* styles go here */
+div {
+  line-height: 10px;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:4]: Use classes for styling purposes and margins instead `br`

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Comment by Dimoshua** [comment]
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_html-form/report/html_report/)

**Pull request closed without merging**